### PR TITLE
bash-completion: update to 2.13.0

### PR DESCRIPTION
**Pull request by i0ntempest**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS x.y
Xcode x.y / Command Line Tools x.y.z

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL in commit message? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

**Review by raimue — APPROVED** [review]
Thanks for the port update. Looks good to me! :+1:

**Comment by raimue** [comment]
Let's wait for the CI checks to finish before merging.

**Comment by i0ntempest** [comment]
@raimue I forgot to include updated patches (just reference updates, no functional changes). Do you want anything else in those patch headers?

File: sysutils/bash-completion/files/patch-xine-add-more-formats.diff
Comment (raimue) [inline sysutils/bash-completion/files/patch-xine-add-more-formats.diff:7]: But this is for version 2.13, isn't it?

**Review by raimue — COMMENTED** [review]
Besides the comment added to the patch file headers, I do not see any actual changes to the patches? That leaves me a bit confused.

File: sysutils/bash-completion/files/patch-xine-add-more-formats.diff
Comment (i0ntempest) [reply sysutils/bash-completion/files/patch-xine-add-more-formats.diff:7]: The patches were changed [here](https://github.com/macports/macports-ports/pull/22795) and you said back then we should indicate they were no longer taken directly from another distribution. Hence I added they were changed for 2.12.

**Pull request merged**
